src/dune_pkg/opam_solver.ml

```diff
@@ -244,30 +242,34 @@ module Context = struct
   ;;
 
   let filter_deps t package filtered_formula =
-    let name = OpamPackage.name package |> Package_name.of_opam_package_name in
     (* Add additional constraints to the formula. This works in two steps.
        First identify all the additional constraints applied to packages which
        appear in the current package's dependency formula. Then each additional
        constnraint is and-ed with the current package's dependency formula. *)
     let filtered_formula =
       OpamFormula.fold_left
         (fun additional_formulae (pkg, _) ->
-          let name = Package_name.of_opam_package_name pkg in
-          match Package_name.Map.find t.constraints name with
+          match
+            let name = Package_name.of_opam_package_name pkg in
+            Package_name.Map.find t.constraints name
```

Comment:
```suggestion
            Package_name.of_opam_package_name pkg
            |> Package_name.Map.find t.constraints name
```
Considering this PR is adding |>, is there a reason not to do this?

Reply:
No objection from me. There's plenty of places where this improvement could be done but hasn't been yet.